doc/source/api.rst

```diff
@@ -5,66 +5,108 @@
 API Reference
 =============
 
-.. currentmodule:: cvpy
+**********
+Annotation
+**********
 
-Biomedical Images
------------------
+Base
+====
+
+Credentials
+-----------
+
+.. currentmodule:: cvpy.annotation.base.Credentials
 
-.. currentmodule:: cvpy.biomedimage.BiomedImage
-    
 .. autosummary::
     :toctree: generated/
 
-    BiomedImage
-    BiomedImage.quantify_sphericity
-    BiomedImage.mask_image
-    BiomedImage.morphological_gradient
+    Credentials
 
-CAS Thread Tuner
-----------------
+Project
+-------
 
-.. currentmodule:: cvpy.utils.CASThreadTuner
+.. currentmodule:: cvpy.annotation.base.Project
 
 .. autosummary::
     :toctree: generated/
 
-    CASThreadTuner
-    CASThreadTuner.tune_thread_count
+    Project
+    Project.as_dict
+    Project.get_annotations
+    Project.post_images
+    Project.resume
+    Project.save
+    Project.to_json
 
+CVAT
+====
 
-CAS Thread Tuner Results
-------------------------
+.. currentmodule:: cvpy.annotation.cvat
 
-.. currentmodule:: cvpy.base.CASThreadTunerResults
+CVAT Project
+------------
+
+.. currentmodule:: cvpy.annotation.cvat.CVATProject
 
 .. autosummary::
     :toctree: generated/
 
-    CASThreadTunerResults
-    CASThreadTunerResults.plot_exec_times
+    CVATProject
+    CVATProject.as_dict
+    CVATProject.from_json
+    CVATProject.get_annotations
+    CVATProject.post_images
+    CVATProject.save
+    CVATProject.resume
+    CVATProject.to_json
+
+CVAT Task
+---------
+
+.. currentmodule:: cvpy.annotation.cvat.CVATTask
+
+.. autosummary::
+    :toctree: generated/
+
+    CVATTask
+    CVATTask.from_dict
 
-Image
------
+CVAT Authentication
+-------------------
 
-.. currentmodule:: cvpy.image.Image
+.. currentmodule:: cvpy.annotation.cvat.CVATAuthenticator
 
 .. autosummary::
     :toctree: generated/
 
-    Image
-    Image.convert_to_CAS_column
-    Image.convert_wide_to_numpy
-    Image.convert_numpy_to_wide
-    Image.fetch_geometry_info
-    Image.fetch_image_array
-    Image.get_image_array_from_row
-    Image.get_image_array
-    Image.get_image_array_const_ctype
-    Image.mask_image
+    CVATAuthenticator
+    CVATAuthenticator.authenticate
+    CVATAuthenticator.generate_cvat_token
+
+******
+Tables
+******
+
+Biomedical Image Table
+======================
+
+.. currentmodule:: cvpy.biomedimage.BiomedImageTable
+    
+.. autosummary::
+    :toctree: generated/
 
+    BiomedImageTable
+    ImageTable.as_dict
+    BiomedImageTable.fetch_image_array
+    BiomedImageTable.fetch_geometry_info
+    BiomedImageTable.from_table
+    ImageTable.has_decoded_images
+    BiomedImageTable.load
+    BiomedImageTable.sphericity
+    BiomedImageTable.morphological_gradient
```

Comment:
We expect users to call ImageTable.from_table() and ImageTable.load() functions. So we can remove the following. 
BiomedImageTable.from_table
BiomedImageTable.load